Podfile

```diff
@@ -50,9 +50,9 @@ def wordpress_ui
 end
 
 def wordpress_kit
-  pod 'WordPressKit', '~> 8.4-beta'
-  # pod 'WordPressKit', git: 'https://github.com/wordpress-mobile/WordPressKit-iOS.git', branch: ''
+  pod 'WordPressKit', '~> 8.5.0-beta.3'
```

Comment:
FYI: This specification constraints WordPressKit to versions compatible with `8.5.0`, including beta. That is the like of `8.5.1-beta` and `8.5.2` but not `8.6.0` and above.

Using `~> MAJOR.MINOR-beta` like before achieves the desired result of keeping the resolved version compatible with all new releases.

Not a big deal because these values change often, but good to keep in mind.